olive/engine/packaging/packaging_generator.py

```diff
@@ -67,10 +67,15 @@ def _package_candidate_models(tempdir, footprint: Footprint, pf_footprint: Footp
         # Copy inference config
         inference_config_path = str(model_dir / "inference_config.json")
         inference_config = pf_footprint.get_model_inference_config(model_id)
```

Comment:
`get_model_inference_config` return `return model_config.get("config", {}).get("inference_settings", None)` 

seems we cannot use `inference_config = inference_config or {"inference_settings": {}}`